website/docusaurus.config.js

```diff
@@ -259,7 +259,7 @@ const siteConfig = {
           showLastUpdateAuthor: true,
           // Equivalent to `enableUpdateTime`.
           showLastUpdateTime: true,
-          routeBasePath: '',
+          routeBasePath: '/',
```

Comment:
this change was needed to for alpha 62 to work, will likely make it mandatory for this path to start with /